setup.py

```diff
@@ -47,53 +46,50 @@ def read(fname):
     'singledispatchmethod; python_version < "3.8.0"',
     'dataclasses; python_version < "3.7.0"',
     'PyYAML>=5.1',
-    'orjson>=3.6.1',    # this is a faster json lib support inf/nan and datetime
-                        # v3.6.1 is the last version supports Python 3.6
+    'orjson>=3.6.1',  # this is a faster json lib support inf/nan and datetime
+    # v3.6.1 is the last version supports Python 3.6
 ]
 
 # extra requirements for biothings.web
 web_extra_requires = [
-    'msgpack>=0.6.1',   # support format=msgpack
+    'msgpack>=0.6.1',  # support format=msgpack
     'sentry-sdk>=1.5.3',  # new sentry package
 ]
 
 # extra requirements to run biothings.hub
 hub_requires = [
-    'beautifulsoup4',   # used in dumper.GoogleDriveDumper
-    'aiocron==1.8',     # setup scheduled jobs
-    'aiohttp==3.8.1',   # elasticsearch requires aiohttp>=3,<4
+    'beautifulsoup4',  # used in dumper.GoogleDriveDumper
+    'aiocron==1.8',  # setup scheduled jobs
+    'aiohttp==3.8.1',  # elasticsearch requires aiohttp>=3,<4
     'asyncssh==2.11.0',  # needs libffi-dev installed (apt-get)
     'pymongo>=4.1.0,<5.0',  # support MongoDB 5.0 since v3.12.0
     'psutil',
-    'jsonpointer',      # for utils.jsonpatch
-    'IPython',          # for interactive hub console
+    'jsonpointer',  # for utils.jsonpatch
+    'IPython',  # for interactive hub console
     'multiprocessing_on_dill',  # can replace pickler in concurrent.futures
-    'dill',             # a pickle alternative with extra object type support
-    'pyinotify; sys_platform == "linux"',        # Linux-only; used in utils.hub.PyInotifyHubReloader
-    'prettytable',      # diff report renderer
-    'sockjs-tornado==1.0.7',   # websocket server for HubServer
+    'dill',  # a pickle alternative with extra object type support
+    'pyinotify; sys_platform == "linux"',  # Linux-only; used in utils.hub.PyInotifyHubReloader
+    'prettytable',  # diff report renderer
+    'sockjs-tornado==1.0.7',  # websocket server for HubServer
     'jsonschema>=2.6.0',
-    'pip',              # auto-install requirements from plugins
+    'pip',  # auto-install requirements from plugins
     # 'pandas==1.0.1',    # json with inf/nan and more to come (replaced by orjson below now)
     # 'orjson>=3.5.2',    # this is a faster json lib support inf/nan and datetime
-    'yapf',             # code reformatter, better results than autopep8
-    'requests-aws4auth',    # aws s3 auth requests for autohub
-    'networkx>=2.1,<2.6',            # datatransform
-    'biothings_client>=0.2.6'   # datatransform (api client)
+    'yapf',  # code reformatter, better results than autopep8
+    'requests-aws4auth',  # aws s3 auth requests for autohub
+    'networkx>=2.1,<2.6',  # datatransform
+    'biothings_client>=0.2.6',  # datatransform (api client)
 ]
 
 # extra requirements to develop biothings
 dev_requires = [
     'pytest',
     'pytest-mock',
+    'pre-commit==2.17.0',
```

Comment:
is `fourmat` also required here now?

Reply:
no it isn't, we define it in the [.pre-commit-config.yaml](https://github.com/newgene/biothings.api/pull/53/files/0bfc9a2cb6451eb269d2aabafd796d2744ecd8b6#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R12)